src/shared/prerendered-app/Intro/style.css

```diff
@@ -118,6 +118,93 @@
   fill: var(--light-blue);
 }
 
+.bottom-wave {
+  position: relative;
+}
+
+.info {
+  background: var(--light-gray);
+  position: relative;
+  padding: 5em 2em;
+  @media (min-width: 1200px) {
+    padding: 5em 0;
+  }
+}
+
+.info-container {
+  max-width: 1000px;
+  margin: 0 auto;
+  opacity: 0;
+  @media screen and (prefers-reduced-motion: no-preference) {
+    transform: translate(0px, 40px);
+    transition: opacity, transform 300ms ease-in;
+  }
+}
+
+.info-container.appear {
+  opacity: 1;
+  @media screen and (prefers-reduced-motion: no-preference) {
+    transform: translate(0px, 0px);
+  }
+}
+
+.info-content {
+  display: grid;
+  align-items: center;
+  grid-template-columns: repeat(auto-fit, minmax(320px, 1fr));
+  gap: 1em;
+}
+
+.dir-rtl {
+  direction: rtl;
```

Comment:
are these `direction`s really needed?

Reply:
Change coming :)

Reply:
@argyleink I switched away from the `direction` but I'm having trouble using `:nth-child` or `:nth-of-type` like your codepen. Maybe I'm missing something? I'm using an alternative atm.

Ideally, my scenario would be:

```css

.info-text-wrapper {
    ...
    &:nth-of-type(even) {
        order: 2;
        justify-self: end;
     }
}
...
.info-img-wrapper {
    ...
    &:nth-of-type(even) {
        order: 1;
        justify-self: start;
     }
}
```

However, (using `nth-of-type` and `nth-child`) the following results:

The `.info-img-wrapper` is selected in all `sections` as it behaves as the `even` child. And the `.info-text-wrapper` is ignored since it is not the `even` child. Something I'm thinking about further...

Reply:
lookin at the DOM and your classes, this is what I have working:

```css
.info:nth-of-type(even) {
  ...
  & .info-img-wrapper {
    grid-column: 1;
    grid-row: 1;
  }

  & .info-text-wrapper {
    text-align: right;
  }
}
```

Reply:
Thanks for the tip, that helped solve it. Let me know what you think :)